src/Sarif.Converters/PluginConverterFactory.cs

```diff
@@ -9,21 +9,21 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Converters
 {
-    public delegate Assembly AssemblyLoadFileDelegate(string path);
+    public delegate Assembly AssemblyLoadFromDelegate(string path);
 
     // Factory class for creating a converter from a specified plug-in assembly.
     internal class PluginConverterFactory : ConverterFactory
     {
         // This field is internal, rather than private, for test purposes.
         internal readonly string pluginAssemblyPath;
 
-        private readonly AssemblyLoadFileDelegate assemblyLoadFileDelegate;
+        private readonly AssemblyLoadFromDelegate assemblyLoadFromDelegate;
 
         internal PluginConverterFactory(
             string pluginAssemblyPath,
-            AssemblyLoadFileDelegate assemblyLoadFileDelegate = null)
+            AssemblyLoadFromDelegate assemblyLoadFromDelegate = null)
         {
-            this.assemblyLoadFileDelegate = assemblyLoadFileDelegate ?? Assembly.LoadFile;
+            this.assemblyLoadFromDelegate = assemblyLoadFromDelegate ?? Assembly.LoadFrom;
```

Comment:
The change from `Assembly.LoadFile` to `Assembly.LoadFrom` is the fix. Everything else is just variable and type renames.